paasta_tools/chronos_tools.py

```diff
@@ -154,6 +154,10 @@ class UnknownChronosJobError(Exception):
     pass
 
 
+class MultipleChronosJobs(Exception):
```

Comment:
Yea, I decided to stop trying to raise at this point and let the callers figure it out and let the function itself always return multiple jobs.